asserts/snap_asserts.go

```diff
@@ -842,6 +847,19 @@ func checkDevelopers(headers map[string]interface{}) (map[string][]*dateRange, e
 			return nil, fmt.Errorf(`"since" %s must be less than or equal to "until"`, what)
 		}
 
+		// Track/check for revocation conflicts.
+		revoked := since.Equal(until)
+		previouslyRevoked, ok := revocationStatus[accountID]
+		if !ok {
+			revocationStatus[accountID] = revoked
+		} else if previouslyRevoked && revoked {
```

Comment:
I would have just comapared revoked != previouslyRevoked and emitted a more generic error mentioning developer-id